app.js

```diff
@@ -35,12 +36,19 @@ app.use((req, res, next) => {
   next();
 });
 app.use(
-  express.static(__dirname + '/node_modules/@actual-app/web/build', {
-    index: false
-  })
+  express.static(
+    path.resolve(__dirname + '/../node_modules/@actual-app/web/build'),
+    {
+      index: false
+    }
+  )
 );
 app.get('/*', (req, res) => {
-  res.sendFile(__dirname + '/node_modules/@actual-app/web/build/index.html');
+  res.sendFile(
+    path.resolve(
+      __dirname + '/../node_modules/@actual-app/web/build/index.html'
+    )
+  );
```

Comment:
Looking at the check failure (which is probably safe to ignore like you did), we could pull the file out into a top level `fs.readFileSync` and then just send back the buffer each time.